styles/XH.scss

```diff
@@ -51,7 +51,7 @@ body.xh-app {
     margin: var(--xh-pad-px) 0;
   }
 
```

Comment:
Seems like this block below could be moved to kit/blueprint/styles.scss since it deals specifically with blueprint overrides.

Reply:
Yes, I agree - would be best to consolidate these styles in one place

Reply:
Moved to kit/blueprint/styles.scss

Reply:
Now removed entirely.  Does not appear to be needed.